### PR TITLE
Add option to set ism

### PR DESCRIPTION
**Pull request by ststkent**

### CODE REVIEW

File: script/commands/warp.ts [hunk 1]
```diff
@@ -90,7 +96,12 @@ function checkConfigType<
   return config.type === tokenType && config.mode === tokenMode;
 }
 
-async function handleCreate(configFile: string) {
+async function handleCreate(configFile: string, cmd: Command) {
+  type Option = {
+    ismAddress?: string;
```
Comment (sampocs) [inline script/commands/warp.ts]: do you have to do the 0x for this one as well?

**Review by sampocs — APPROVED** [review]
lgtm!

File: script/commands/warp.ts (hunk 1 above)
Comment (ststkent) [reply script/commands/warp.ts]: i don't see an error but I don't think it'd hurt!

Comment (ststkent) [reply script/commands/warp.ts]: added, merging

**Pull request merged**